internal/setup/setup_test.go

```diff
@@ -1063,6 +1063,174 @@ var setupTests = []setupTest{{
 		`,
 	},
 	relerror: `invalid slice definition filename: "a.yaml"`,
+}, {
+	summary: "Specify generate: manifest",
+	input: map[string]string{
+		"slices/mydir/mypkg.yaml": `
+			package: mypkg
+			slices:
+				myslice:
+					contents:
+						/path/**: {generate: "manifest"}
```

Comment:
[Nit]: Please use consistent naming, it does not make sense to have one package `mypkg` with `myslice` and `path` and the other `foo` `bar` and `foo`. I would suggest using `mypkg1` and `mypkg2`.